config/sunspot.yml

```diff
@@ -1,24 +1,23 @@
----
 production:
   solr:
-    hostname: pad.publiclab.org
+    hostname: localhost
     port: 8983
-    read_timeout: 120
     log_level: WARNING
-    path: "/solr/default"
-  disabled: true
```

Comment:
I remember why, now--was working on issues getting my testing complete.  didn't realize this was a tracked file.

I'll fix and resubmit.  Really appreciate the help!